assets/css/base/_base.scss

```diff
@@ -846,9 +846,7 @@ select,
 textarea {
 	font-size: 100%; /* Corrects font size not being inherited in all browsers */
 	margin: 0; /* Addresses margins set differently in IE6/7, F3/4, S5, Chrome */
-	vertical-align: baseline; /* Improves appearance and consistency in all browsers */
-	/* stylelint-disable */
-	*vertical-align: middle; /* Improves appearance and consistency in all browsers */
+	vertical-align: middle; /* Improves appearance and consistency in all browsers */
 	/* stylelint-enable */
```

Comment:
When removing `/* stylelint-disable */`, `/* stylelint-enable */` should also be removed.